<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1232</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="init_pic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>350</width>
      <height>350</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="error_pic">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>350</width>
      <height>350</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="result_pic">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>350</width>
      <height>350</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="object_pic">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="obj_rot_r_pic">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>140</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="obj_rot_l_pic">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>260</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Найти</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="snr_box">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>380</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QRadioButton" name="rot_box">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>380</y>
      <width>99</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Left/Right</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>380</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SNR</string>
    </property>
   </widget>
   <widget class="QLabel" name="fghjk">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>380</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>k</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="koeff">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>380</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="fghjk_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>370</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>x_ref:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="x_ref_box">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>370</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="maximum">
     <number>250</number>
    </property>
   </widget>
   <widget class="QLabel" name="fghjk_3">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>400</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>y_ref:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="y_ref_box">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>400</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="maximum">
     <number>250</number>
    </property>
   </widget>
   <widget class="QLabel" name="fghjk_4">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>370</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>x:</string>
    </property>
   </widget>
   <widget class="QLabel" name="fghjk_5">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>400</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>y:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="x_box">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>370</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="maximum">
     <number>250</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="y_box">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>400</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="maximum">
     <number>250</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1232</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
